ibis/backends/tests/conftest.py

```diff
@@ -192,6 +192,13 @@ def df(alltypes):
     return alltypes.execute()
 
 
+@pytest.fixture(scope='session')
+def pandas_df(backend, alltypes):
+    if backend.name() == "dask":
+        return alltypes.execute().compute()
```

Comment:
This is a bit weird. I think the interface of "execute" defines that it returns a pd.DataFrame/pd.Series. I feel it's not so great to change the interface here. Perhaps we should define a method on Dask backend to return a dd.DataFrame that is not called execute().

For example, PySpark backend seperates these two methods:
https://github.com/ibis-project/ibis/blob/master/ibis/backends/pyspark/client.py#L25

Can be a follow up

Reply:
The dask backend never returns a `pd.DataFrame` or `pd.Series` on execute (nor do I think it should). It always gives you back a dask task graph you are required to run `compute` on. This creates a problem for a few tests, particularly in `test_aggregations` that expect the expected test case to be a pandas `DataFrame` and support all the pandas functions.